### PR TITLE
Add hint for location of app.yml file

### PR DESCRIPTION
**Pull request by nischayn22**
I had to google for the file location and landed [here](https://meta.discourse.org/t/how-to-edit-app-yml-file/20569). So adding it directly in the documentation.

### CODE REVIEW

File: docs/INSTALL-cloud.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ This will generate an `app.yml` configuration file on your behalf, and then kick
 
 - For proper email deliverability, add valid SPF and DKIM records in your DNS. See your email provider instructions for specifics.
 
-If you need to change or fix your email settings after bootstrapping, edit your `app.yml` file and `./launcher rebuild app`, otherwise your changes will not take effect.
+If you need to change or fix your email settings after bootstrapping, edit your `app.yml` file (vi containers/app.yml) and `./launcher rebuild app`, otherwise your changes will not take effect.
```
Comment (tgxworld) [inline docs/INSTALL-cloud.md]: `/var/discourse/containers/app.yml instead`?

**Comment by tgxworld** [comment]
Just a tip, you can add `[ci skip]` to your commit message so that we don't run the tests for non code related commits.

**Comment by tgxworld** [comment]
:+1: Please squash your commits and I'll merge. Thanks!

**Comment by nischayn22** [comment]
Will do in a while, see no option to do it from the UI in github.

**Comment by SamSaffron** [comment]
I am closing this, recommending users use `vi` to edit a file in a readme is not needed for people who know how to use vi and is a disaster for those who don't :) 

@coding-horror  feel free to edit install cloud to clarify if you see fit.

**Pull request closed without merging**